Using KUMA_API_SERVER_BASE_PATH can cause a panic

### What happened?

`KUMA_API_SERVER_BASE_PATH=ui kuma-cp run` using the latest binary from the install script causes a panic:

```
panic: parsing "ui": at offset 0: host/path missing /

goroutine 1 [running]:
net/http.(*ServeMux).register(...)
        net/http/server.go:2797
net/http.(*ServeMux).HandleFunc(0x1400007401a?, {0x1400007401a?, 0x102caef14?}, 0x1036a1a08?)
        net/http/server.go:2771 +0x78
github.com/emicklei/go-restful/v3.(*Container).addHandler(0x140004d8bd0, 0x140008dcc40, 0x140008dcb60)
        github.com/emicklei/go-restful/v3@v3.12.1/container.go:132 +0x1e8
github.com/emicklei/go-restful/v3.(*Container).Add(0x140004d8bd0, 0x140008dcc40)
        github.com/emicklei/go-restful/v3@v3.12.1/container.go:107 +0x1b8
github.com/kumahq/kuma/pkg/api-server.NewApiServer({0x10373a5e0, 0x1400094fef0}, {0x103708a00, 0x14000a89440}, {0x1400063e008, 0x30, 0x49}, 0x14000b12e08, {0x0, 0x0, ...})
        github.com/kumahq/kuma/pkg/api-server/server.go:180 +0xc80
github.com/kumahq/kuma/pkg/api-server.SetupServer({0x10373a5e0, 0x1400094fef0})
        github.com/kumahq/kuma/pkg/api-server/server.go:469 +0x3f0
github.com/kumahq/kuma/app/kuma-cp/cmd.newRunCmdWithOpts.func1(0x14000ae1b08, {0x1025d6460?, 0x4?, 0x1025d6464?})
        github.com/kumahq/kuma/app/kuma-cp/cmd/run.go:136 +0x4a0
github.com/spf13/cobra.(*Command).execute(0x14000ae1b08, {0x10592d640, 0x0, 0x0})
        github.com/spf13/cobra@v1.8.1/command.go:985 +0x834
github.com/spf13/cobra.(*Command).ExecuteC(0x14000ae1808)
        github.com/spf13/cobra@v1.8.1/command.go:1117 +0x344
github.com/spf13/cobra.(*Command).Execute(...)
        github.com/spf13/cobra@v1.8.1/command.go:1041
github.com/kumahq/kuma/app/kuma-cp/cmd.Execute()
        github.com/kumahq/kuma/app/kuma-cp/cmd/root.go:83 +0x20
main.main()
        github.com/kumahq/kuma/app/kuma-cp/main.go:6 +0x1c
```

--- a/pkg/api-server/server.go
+++ b/pkg/api-server/server.go
@@ -131,9 +131,13 @@ func NewApiServer(
 	// We create a WebService and set up resources endpoints and index endpoint instead of creating WebService
 	// for every resource like /meshes/{mesh}/traffic-permissions, /meshes/{mesh}/traffic-log etc.
 	// because go-restful detects it as a clash (you cannot register 2 WebServices with path /meshes/)
+	apiBasePath := cfg.ApiServer.BasePath
+	if !strings.HasPrefix(apiBasePath, "/") {
+		apiBasePath = "/" + apiBasePath
+	}
 	ws := new(restful.WebService)
 	ws.
-		Path(cfg.ApiServer.BasePath).
+		Path(apiBasePath).
 		Consumes(restful.MIME_JSON).
 		Produces(restful.MIME_JSON)
 

